### PR TITLE
Polish release notes 1.x

### PR DESCRIPTION
**Pull request by Jay-o-Way**
#8352

### CODE REVIEW

**Comment by Jay-o-Way** [comment]
Hold on, i saw some things i have to fix

**Review by mairaw — CHANGES_REQUESTED** [review]
LGTM. Left a few comments. Didn't duplicate the same comment across all occcurrences.

File: release-notes/1.1/1.1-commits.md [hunk 1]
```diff
@@ -231,15 +231,15 @@
...
 - [`[bfcd630]`](https://github.com/dotnet/coreclr/commit/bfcd630) Specify packages.builds by default in build-packages command
 - [`[a641d3a]`](https://github.com/dotnet/coreclr/commit/a641d3a) Makes FCallMemcpy unwindable (#6753)
-- [`[c037777]`](https://github.com/dotnet/coreclr/commit/c037777) Apply correct symbolic name _ARM_ instead of _TARGET_ARM_ (#6756)
+- [`[c037777]`](https://github.com/dotnet/coreclr/commit/c037777) Apply correct symbolic name \_ARM_instead of \_TARGET_ARM_ (#6756)
 - [`[a7bac8b]`](https://github.com/dotnet/coreclr/commit/a7bac8b) printing dumpling url at upload time.
 - [`[b6be3a0]`](https://github.com/dotnet/coreclr/commit/b6be3a0) Fix build break from new alignments
 - [`[62f246c]`](https://github.com/dotnet/coreclr/commit/62f246c) Adding new generics tests (#6709)
 - [`[6e782a6]`](https://github.com/dotnet/coreclr/commit/6e782a6) Add deserialization ctors to mscorlib ref assembly
-- [`[91de2ce]`](https://github.com/dotnet/coreclr/commit/91de2ce) Record expr location for opqaue value numbers
+- [`[91de2ce]`](https://github.com/dotnet/coreclr/commit/91de2ce) Record expr location for opaque value numbers
```
Comment (mairaw) [inline release-notes/1.1/1.1-commits.md:239]:
```suggestion
- [`[91de2ce]`](https://github.com/dotnet/coreclr/commit/91de2ce) Record expr location for opqaue value numbers
```

File: release-notes/1.1/1.1-commits.md [hunk 2]
```diff
@@ -231,15 +231,15 @@
...
 - [`[ca04119]`](https://github.com/dotnet/coreclr/commit/ca04119) Arm64 Sos fixes for !verify heap !u !u -gcinfo
 - [`[607816f]`](https://github.com/dotnet/coreclr/commit/607816f) update GC shadow heap as part of writebarrier jit helper
-- [`[ab1d328]`](https://github.com/dotnet/coreclr/commit/ab1d328) Fix issue https://github.com/dotnet/coreclr/issues/6600
+- [`[ab1d328]`](https://github.com/dotnet/coreclr/commit/ab1d328) Fix issue <https://github.com/dotnet/coreclr/issues/6600>
```
Comment (mairaw) [inline release-notes/1.1/1.1-commits.md:242]:
```suggestion
- [`[ab1d328]`](https://github.com/dotnet/coreclr/commit/ab1d328) Fix issue https://github.com/dotnet/coreclr/issues/6600
```

File: release-notes/1.1/1.1-commits.md [hunk 3]
```diff
@@ -415,7 +415,7 @@
 - [`[fe4ac43]`](https://github.com/dotnet/coreclr/commit/fe4ac43) RyuJIT/x86: implement tailcall via helper
 - [`[59f169e]`](https://github.com/dotnet/coreclr/commit/59f169e) Fix #6125 (#6432)
 - [`[087e750]`](https://github.com/dotnet/coreclr/commit/087e750) GC update from CoreRT (#6429)
-- [`[ac01e27]`](https://github.com/dotnet/coreclr/commit/ac01e27) Add link to https://graphics.stanford.edu/~seander/bithacks.html#ZeroInWord (#6430)
+- [`[ac01e27]`](https://github.com/dotnet/coreclr/commit/ac01e27) Add link to <https://graphics.stanford.edu/~seander/bithacks.html#ZeroInWord> (#6430)
```
Comment (mairaw) [inline release-notes/1.1/1.1-commits.md:418]:
```suggestion
- [`[ac01e27]`](https://github.com/dotnet/coreclr/commit/ac01e27) Add link to https://graphics.stanford.edu/~seander/bithacks.html#ZeroInWord (#6430)
```

File: release-notes/1.1/1.1-commits.md [hunk 4]
```diff
@@ -847,7 +847,7 @@
 - [`[7a47353]`](https://github.com/dotnet/coreclr/commit/7a47353) Linux/ARM: Revert -O1 to -O3 for Linux/ARM Release Build. (#5598)
 - [`[c413a60]`](https://github.com/dotnet/coreclr/commit/c413a60) Fixing perf runs on helix (#5102)
 - [`[8e41205]`](https://github.com/dotnet/coreclr/commit/8e41205) software ww fix
-- [`[68a832d]`](https://github.com/dotnet/coreclr/commit/68a832d) Fix build issue http://buildstatus/Issues/Issues.aspx?iid=802303
+- [`[68a832d]`](https://github.com/dotnet/coreclr/commit/68a832d) Fix build issue <http://buildstatus/Issues/Issues.aspx?iid=802303>
```
Comment (mairaw) [inline release-notes/1.1/1.1-commits.md:850]:
```suggestion
- [`[68a832d]`](https://github.com/dotnet/coreclr/commit/68a832d) Fix build issue http://buildstatus/Issues/Issues.aspx?iid=802303
```

File: release-notes/1.1/1.1-commits.md [hunk 5]
```diff
@@ -1359,7 +1359,7 @@
 - [`[cea2b84]`](https://github.com/dotnet/corefx/commit/cea2b84) Add full-version package dependency verification.
 - [`[ce6561a]`](https://github.com/dotnet/corefx/commit/ce6561a) Fix bug in property System.Console.ConsolePal.Windows.WIndowTop
 - [`[a118b9a]`](https://github.com/dotnet/corefx/commit/a118b9a) Revving xunit perf packages to fix perf test runs (#10869)
-- [`[9130f83]`](https://github.com/dotnet/corefx/commit/9130f83) Fix for issue https://github.com/dotnet/wcf/issues/1340 (DataContract special-handling of KeyValuePair generic type): data contracts for the adapter and the original type do not get the same StableName (contract name)
+- [`[9130f83]`](https://github.com/dotnet/corefx/commit/9130f83) Fix for issue <https://github.com/dotnet/wcf/issues/1340> (DataContract special-handling of KeyValuePair generic type): data contracts for the adapter and the original type do not get the same StableName (contract name)
```
Comment (mairaw) [inline release-notes/1.1/1.1-commits.md:1362]:
```suggestion
- [`[9130f83]`](https://github.com/dotnet/corefx/commit/9130f83) Fix for issue https://github.com/dotnet/wcf/issues/1340 (DataContract special-handling of KeyValuePair generic type): data contracts for the adapter and the original type do not get the same StableName (contract name)
```

File: release-notes/1.1/1.1-commits.md [hunk 6]
```diff
@@ -2427,7 +2427,7 @@
 - [`[a4cef6d]`](https://github.com/dotnet/cli/commit/a4cef6d) Updating NuGet to beta2-1484
 - [`[1a9b16a]`](https://github.com/dotnet/cli/commit/1a9b16a) Adding dotnet-core feed to the NuGet.config since that has the stable versions of the .NET runtime.
 - [`[f25f047]`](https://github.com/dotnet/cli/commit/f25f047) Updating CoreFX, CoreCLR, Roslyn, CoreSetup to stable versions.
-- [`[dfe99ca]`](https://github.com/dotnet/cli/commit/dfe99ca) Port https://github.com/dotnet/core-setup/pull/159 to CLI
+- [`[dfe99ca]`](https://github.com/dotnet/cli/commit/dfe99ca) Port <https://github.com/dotnet/core-setup/pull/159> to CLI
```
Comment (mairaw) [inline release-notes/1.1/1.1-commits.md:2430]:
```suggestion
- [`[dfe99ca]`](https://github.com/dotnet/cli/commit/dfe99ca) Port https://github.com/dotnet/core-setup/pull/159 to CLI
```

File: release-notes/1.1/1.1-commits.md [hunk 7]
```diff
@@ -2131,7 +2131,7 @@
 
 ## WCF
 
-- [`[674312d]`](https://github.com/dotnet/wcf/commit/674312d) http://dotnet-ci.cloudapp.net -> https://ci.dot.net
+- [`[674312d]`](https://github.com/dotnet/wcf/commit/674312d) <http://dotnet-ci.cloudapp.net> -> <https://ci.dot.net>
```
Comment (mairaw) [inline release-notes/1.1/1.1-commits.md:2134]:
```suggestion
- [`[674312d]`](https://github.com/dotnet/wcf/commit/674312d) http://dotnet-ci.cloudapp.net -> https://ci.dot.net
```

File: release-notes/1.1/1.1-commits.md [hunk 8]
```diff
@@ -2511,7 +2511,7 @@
 - [`[351255d]`](https://github.com/dotnet/cli/commit/351255d) Updating CoreSetup to rc3-004373
 - [`[74a9594]`](https://github.com/dotnet/cli/commit/74a9594) Small fixes to update-dependencies.
 - [`[64508f0]`](https://github.com/dotnet/cli/commit/64508f0) fixes #2408 stop using corehost to activate projectdependencycommands
-- [`[06047ee]`](https://github.com/dotnet/cli/commit/06047ee) Update only the ubuntu.14.04 dockerfile from https://github.com/dotnet/core-setup/pull/58
+- [`[06047ee]`](https://github.com/dotnet/cli/commit/06047ee) Update only the ubuntu.14.04 dockerfile from <https://github.com/dotnet/core-setup/pull/58>
```
Comment (mairaw) [inline release-notes/1.1/1.1-commits.md:2514]:
```suggestion
- [`[06047ee]`](https://github.com/dotnet/cli/commit/06047ee) Update only the ubuntu.14.04 dockerfile from https://github.com/dotnet/core-setup/pull/58
```

File: release-notes/1.1/1.1.5.md [hunk 9]
```diff
@@ -73,7 +73,7 @@ The [.NET Core Docker images](https://hub.docker.com/r/microsoft/dotnet/) have b
 * [`[2278610]`](https://github.com/dotnet/coreclr/commit/2278610) Fix resource lookup recursion issue (#13945)
 * [`[ad68ca9]`](https://github.com/dotnet/coreclr/commit/ad68ca9) Remove EOL openSuSE 42.1 (#13691)
 * [`[0cb88b8]`](https://github.com/dotnet/coreclr/commit/0cb88b8) Remove FreeBSD 10.1/OpenSuSE 13.2 and Fedora 23 (#13635)
-* [`[0ac7078]`](https://github.com/dotnet/coreclr/commit/0ac7078) Port of https://github.com/dotnet/coreclr/pull/12795 to release/1.1.0 (#12942)
+* [`[0ac7078]`](https://github.com/dotnet/coreclr/commit/0ac7078) Port of <https://github.com/dotnet/coreclr/pull/12795> to release/1.1.0 (#12942)
```
Comment (mairaw) [inline release-notes/1.1/1.1.5.md:76]:
```suggestion
* [`[0ac7078]`](https://github.com/dotnet/coreclr/commit/0ac7078) Port of https://github.com/dotnet/coreclr/pull/12795 to release/1.1.0 (#12942)
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Let's keep commit messages as-is unless display is messed up without the proper formatting.

**Comment by Jay-o-Way** [comment]
PTAL

**Review by mairaw — APPROVED** [review]

**Pull request merged**